pkg/flag/flag.go

```diff
@@ -39,6 +46,10 @@ const (
 	ProviderTypeRedHat            = "redhat"
 	ProviderTypeCertified         = "certified"
 
+	AnnotationSupportedVersions = "pipeline.openshift.io/supported-versions"
+	LabelEnvironmentType        = "pipeline.openshift.io/type"
+	LabelPipelineRuntime        = "pipeline.openshift.io/runtime"
+
```

Comment:
for better context, let us add the word `Pipeline` in Supported versions and enviorment as well
eg: `LabelPipelineEnviormentType`

Reply:
This actually makes sense, will update this. Thank you